snntorch/export_nir.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional
+import torch
+import nir
+import numpy as np
+import nirtorch
```

Comment:
we are now adding a dependency on nirtorch. is this fine?

Reply:
just saw that nirtorch is already in the `setup.py` requirements, so I'm assuming this is no problem

Reply:
Yep it's already there, not a problem!